File: examples/vehicle/lookup/node/lookupVehicle.js
const { VehicleService } = require("m3o/vehicle");

// Lookup a UK vehicle by it's registration number
async function lookupVehicle() {
  let vehicleService = new VehicleService(process.env.MICRO_API_TOKEN);
  let rsp = await vehicleService.lookup({
    registration: "LC60OTA",
  });
  console.log(rsp);
}

lookupVehicle();

File: examples/go/vehicle/lookup/lookupVehicle.go
package example

import (
	"fmt"
	"os"

	"github.com/micro/services/clients/go/vehicle"
)

// Lookup a UK vehicle by it's registration number
func LookupVehicle() {
	vehicleService := vehicle.NewVehicleService(os.Getenv("MICRO_API_TOKEN"))
	rsp, err := vehicleService.Lookup(&vehicle.LookupRequest{
		Registration: "LC60OTA",
	})
	fmt.Println(rsp, err)
}
